rbac/providers/ldap/ldap_sync.py

```diff
@@ -107,9 +107,9 @@ def insert_to_db(data_dict, data_type):
             "timestamp": datetime.now().replace(tzinfo=timezone.utc).isoformat(),
             "provider_id": LDAP_DC
         }
-        r.table("inbound_queue").insert(inbound_entry).run()
+        r.table("queue_inbound").insert(inbound_entry).run()
 
-    print("Inserted %s records into inbound_queue." % len(data_dict))
+    print("Inserted %s records into queue_inbound." % len(data_dict))
```

Comment:
LOGGER, not print.

Reply:
Meh why not, as long as I'm in this file. Replaced usages of print with logger